docs/tutorials/quickstart.md

```diff
@@ -55,41 +55,28 @@ The CMAKE_INSTALL_PREFIX variable must be set to a path with no spaces in it, be
 
 ## To build OTIO for Python development:
 
-### Linux
+**Note** You are highly encouraged to use virtual environements to develop OTIO.
```

Comment:
typo: environments
suggest:
**Note** The use of virtual environments is highly encouraged for OTIO development.

Reply:
environements: The merge of the french word `environnements` and the english word `environments` :upside_down_face: 

I will fix that and I will also rephrase as you suggested. Your suggestion looks better than mine.